<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config</class>
 <widget class="QDialog" name="Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QToolButton" name="shaderButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Shader</string>
   </property>
   <property name="autoRepeatDelay">
    <number>300</number>
   </property>
   <property name="autoRepeatInterval">
    <number>100</number>
   </property>
   <property name="popupMode">
    <enum>QToolButton::InstantPopup</enum>
   </property>
  </widget>
  <widget class="QLabel" name="shaderLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Shader</string>
   </property>
  </widget>
  <widget class="QLabel" name="acceleratorLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Acceleration Structure</string>
   </property>
  </widget>
  <widget class="QToolButton" name="acceleratorButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Accelerator</string>
   </property>
   <property name="popupMode">
    <enum>QToolButton::InstantPopup</enum>
   </property>
  </widget>
  <widget class="QLabel" name="sceneLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Scene</string>
   </property>
  </widget>
  <widget class="QToolButton" name="sceneButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Scene</string>
   </property>
   <property name="popupMode">
    <enum>QToolButton::InstantPopup</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="sppSpinBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>151</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="sppLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Samples per pixel</string>
   </property>
  </widget>
  <widget class="QLabel" name="splLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Samples per light</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="splSpinBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>151</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Config</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Config</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>shaderButton</sender>
   <signal>triggered(QAction*)</signal>
   <receiver>Config</receiver>
   <slot>selected_shader(QAction*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acceleratorButton</sender>
   <signal>triggered(QAction*)</signal>
   <receiver>Config</receiver>
   <slot>selected_accelerator(QAction*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sceneButton</sender>
   <signal>triggered(QAction*)</signal>
   <receiver>Config</receiver>
   <slot>selected_scene(QAction*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sppSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Config</receiver>
   <slot>selected_spp(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>splSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Config</receiver>
   <slot>selected_spl(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selected_shader(QAction*)</slot>
  <slot>selected_accelerator(QAction*)</slot>
  <slot>selected_scene(QAction*)</slot>
  <slot>selected_spp(int)</slot>
  <slot>selected_spl(int)</slot>
 </slots>
</ui>
